README.md

```
@@ -260,7 +262,19 @@ public class FieldExample
 ### Email Management
 
-Send transactional emails through Bento:
+> [!TIP] > **Recommended Approach**: Instead of sending emails directly via API, consider using **Events + Flows** for better email management. This approach allows you to:
+>
+> -   Use visual email templates created in Bento's editor
+> -   Set up automatic triggers and delays
+> -   Add tags and track subscriber behavior
+> -   Implement rate limiting and delivery optimization
+> -   Get detailed analytics and reports
+>
+> See the [Building the Flow guide](https://docs.bentonow.com/deliverability/double_optin#building-the-flow) and [Event-Triggered Email Example](#event-triggered-email-with-flows) below for the recommended pattern.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)